### DISCUSSION TITLE
Welcome to fastapi-sso Discussions!

### DISCUSSION THREAD

**Discussion by tomasvotava** [tomasvotava/fastapi-sso#108] [Announcements]
## 👋 Welcome!

Let's use this place to bring ideas and ask questions 🤓 

Would be awesome we could keep it polite 🙏 

What are you using `fastapi-sso` for?

**Comment by Ilyes-git** [comment]
Hello @tomasvotava ,

I want to thank you for the good work you do to provide this package, well done 👌

I personally use it to connect to Microsoft within my organization.

By the way, I have a question, is there a way to restrict access to endpoints from the swagger, and make the authorize button functional? So, only if you are authenticated with the SSO, you will be able to issue requests.

Or maybe it's already done, but I haven't managed to do it?

Thank you 

Ilyes

**Reply by tomasvotava** [reply]
Hi!

Thanks for reaching out here, I am always glad the library is being used!

Regarding your questions:

- You can get the lock in your swagger docs to work, but you need to implement authorization yourself. `fastapi-sso` only helps you authenticate the user, basically tells you who they are. See [this guide](https://tomasvotava.github.io/fastapi-sso/how-to-guides/use-with-fastapi-security/) for more and feel free to let me know if the example should be extended.
- You can hide your docs entirely when creating the `FastAPI` instance. Just pass `docs_url=None` as seen [here](https://fastapi.tiangolo.com/tutorial/metadata/#docs-urls).

Let me know if there's anything I can do 👌

**Reply by Ilyes-git** [reply]
Thank you verry much :)
